src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## Unreleased
+
+* BUGFIX: Fix issue `merge` command generates SARIF with multiple runs [#2488](https://github.com/microsoft/sarif-sdk/pull/2488)
```

Comment:
>BUGFIX [](http://example.com/codeflow?start=2&length=6)

There's no reference in the release notes to the invalid characters issue.

Reply:
Has someone built/run your tests and examined code coverage? Do you have any information for us on the completeness of code & data coverage for your tests?

Reply:
I ran code coverage analysis in VS. All code paths I made change were covered. In term of `MergeCommand` coverage, it was 84%, added couple test cases its now 94%